src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetMaxByteCount.cs

```diff
@@ -2,27 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
 using Xunit;
 
 namespace System.Text.Tests
 {
     public class ASCIIEncodingGetMaxByteCount
     {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
```

Comment:
I agree with @stephentoub if it is not really random data then we hardcode the values as there is no value of pretending to be random. but I am still thinking it would be better to fix RandomDataGenerator to produce real random data (forcing it to use random seed).